alcs-frontend/src/app/features/application/decision/decision.component.html

```diff
@@ -25,29 +25,49 @@ <h3>Decision</h3>
     </div>
     <div>
       <div class="subheading2">Decision Outcome</div>
-      {{ mapCodeToLabel(decision.outcome) }}
+      {{ decision.outcome.label }}
     </div>
     <div></div>
     <div>
-      <div class="subheading2">Audit Date</div>
-      <span *ngIf="decision.auditDate">{{ decision.auditDate | momentFormat }}</span>
-      <span *ngIf="!decision.auditDate">
-        <app-inline-datepicker (save)="onSaveAuditDate(decision.uuid, $event)"></app-inline-datepicker>
-      </span>
+      <div class="subheading2">Decision Maker</div>
+      {{ decision.decisionMaker?.label }}
     </div>
+    <div>
+      <div *ngIf="decision.ceoCriterion" class="subheading2">CEO Criterion</div>
+      {{ decision.ceoCriterion?.label }}
+      <ng-container *ngIf="decision.ceoCriterion?.code === 'MODI'">
+        {{ decision.isTimeExtension ? '- Time extension' : '- Other' }}
+      </ng-container>
```

Comment:
Is it because decision.isTimeExtension can be null/undefined ?

Reply:
Ill be honest Im not 100% sure anymore, I could replace this with decision.isTimeExtension !== null for the *ngIf.

Reply:
Changed to a null check.